### PR TITLE
Feature/FormWeekday

### PR DESCRIPTION
**Pull request by cgenardini**
## Describe your changes
Created Weekdays form with optional hint text. Form uses useForm/register for state management. 

## Issue ticket number and link

Component: Form.Weekday

https://tripleten-apiary.atlassian.net/browse/DROP-8?atlOrigin=eyJpIjoiNzc1ODk1ZGMzNDMxNDNmMDhiOTliYWI5NDgyZjM5ZWMiLCJwIjoiaiJ9

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [x] I have made any necessary updates to Storybook to accompany these changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: src/components/formWeekday/formWeekday.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      setCurrentSelectedDays(currentSelectedDays.filter((d) => d != day));
+    } else {
+      setCurrentSelectedDays([...currentSelectedDays, day]);
+    }
+  };
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <div className="max-w-pageContent mx-3 relative font-text">
+        <label className="">{title}</label>
```
Comment (Dillona25) [inline src/components/formWeekday/formWeekday.tsx:40]: I would add the proper font to all of the text in this component. Figma calls fore Rubik which would be ```font-text```

File: src/components/formWeekday/formWeekday.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                {...register("selectedDays")}
+                checked={currentSelectedDays.includes(day)}
+                type="checkbox"
+                value={day}
+                className="sr-only"
+                onChange={() => handleCheck(day)}
+              />
+              <label
+                htmlFor={day}
+                className={`flex items-center h-10 w-10 rounded-full border-black/30 border flex justify-center items-center text-sm ${
```
Comment (Dillona25) [inline src/components/formWeekday/formWeekday.tsx:55]: Very well-written CSS and log here 😃

File: src/components/formWeekday/formWeekday.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        <div className="flex justify-between mt-3">
+          {days.map((day) => (
+            <div key={day} className={`relative`}>
+              <input
+                id={day}
+                {...register("selectedDays")}
+                checked={currentSelectedDays.includes(day)}
+                type="checkbox"
+                value={day}
+                className="sr-only"
```
Comment (Dillona25) [inline src/components/formWeekday/formWeekday.tsx:50]: Add the ```font-text``` to the weekday bubbles here to get the Rubik font

File: src/components/formWeekday/formWeekday.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                    ? "text-white bg-black border-black"
+                    : "text-black/60"
+                }`}
+              >
+                {day}
+              </label>
+            </div>
+          ))}
+        </div>
+        {hintText && <span className="text-black/60 text-sm">{hintText}</span>}
```
Comment (Dillona25) [inline src/components/formWeekday/formWeekday.tsx:66]: Font needed here as well

**Review by Dillona25 — CHANGES_REQUESTED** [review]
See comments for needed fixes. Component looks great otherwise!

File: src/components/formWeekday/formWeekday.tsx (hunk 1 above)
Comment (cgenardini) [reply src/components/formWeekday/formWeekday.tsx:40]: I added it on the parent <div> wrapper so I wouldn't have to re-write for each component. Do you think I should add the font-text individually?

**Review by Dillona25 — APPROVED** [review]
Looks great! Go ahead and move this to "final review" column in Jira so Josh can perform the final review!

**Pull request closed without merging**